src/plugins/intel_gpu/src/runtime/ocl/ocl_device.cpp

```diff
@@ -274,8 +274,13 @@ device_info init_device_info(const cl::Device& device) {
 
 bool does_device_support(int32_t param, const cl::Device& device) {
     cl_device_unified_shared_memory_capabilities_intel capabilities;
-    auto err = clGetDeviceInfo(device.get(), param, sizeof(cl_device_unified_shared_memory_capabilities_intel), &capabilities, NULL);
-    if (err) throw std::runtime_error("[CLDNN ERROR]. clGetDeviceInfo error " + std::to_string(err));
+    try {
+        auto err = clGetDeviceInfo(device.get(), param, sizeof(cl_device_unified_shared_memory_capabilities_intel), &capabilities, NULL);
+        if (err) throw std::runtime_error("[CLDNN ERROR]. clGetDeviceInfo error " + std::to_string(err));
```

Comment:
Looks like it is redundant because the exception is already being checked by try catch block?

Reply:
reverted.